src/routes/api/commentRoutes.js

```diff
@@ -1,14 +1,23 @@
 import { Router } from 'express';
 
-import commentControllers from '../../controllers/commentControllers';
-import commentValidation from '../../validation/commentValidation';
-import userMiddlewares from '../../middlewares/userMiddlewares';
+import {
+	addComment,
+	getComments,
+	deleteComment,
+} from '../../controllers/commentControllers';
+import {
+	confirmUserEmail,
+	verifyTravelOwner,
+	verifyCommentOwner,
+} from '../../middlewares/userMiddlewares';
+import {
+	validateComment,
+	validateResult,
+} from '../../validation/commentValidation';
 import tokenMiddleware from '../../middlewares/tokenMiddleware';
 
 const route = Router();
-const { addComment, getComments } = commentControllers;
-const { validateComment, validateResult } = commentValidation;
-const { confirmUserEmail } = userMiddlewares;
+
 const { getToken, verifyToken } = tokenMiddleware;
```

Comment:
Is there any reason why you didn't do the same thing as you did for userMiddlewares?